tests/test_db_utils_edge_cases.py

```
@@ -388,6 +389,654 @@ def test_database_initialization_partial_failure(self):
+
+            # Track calls to shutil.move to simulate sidecar failure
+            call_count = [0]
+
+            def mock_move(src, dst):
+                call_count[0] += 1
+                if call_count[0] == 1:
+                    # First call - move main db successfully
+                    shutil.move(src, dst)
+                elif call_count[0] == 2:
+                    # Second call - sidecar move fails
+                    raise OSError("Sidecar move failed")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)